packages/cli/src/sso/saml/routes/saml.controller.ee.ts

```diff
@@ -0,0 +1,48 @@
+import express from 'express';
+import { issueCookie } from '@/auth/jwt';
+import { AuthError } from '@/ResponseHelper';
+import { samlEnabledMiddleware } from '../middleware/samlEnabledMiddleware';
+import { SamlService } from '../saml.service.ee';
+import { LoggerProxy } from 'n8n-workflow';
+import { SamlUrls } from '../constants';
+
+/**
+ * SSO Endpoints that are protected by samlEnabledMiddleware require
+ * both isSamlEnabled and isSamlCurrentAuthenticationMethod to be true
+ */
+
+export const samlController = express.Router();
+samlController.use(samlEnabledMiddleware);
+
+/**
+ * POST /sso/saml/acs
+ * Assertion Consumer Service endpoint
+ */
+samlController.post(SamlUrls.acs, async (req: express.Request, res: express.Response) => {
```

Comment:
Some providers might redirect using GET so we should also support this method. I think authentik allows you to change to redirect and then it'll be a GET request.

![Screenshot 2023-02-23 at 17 22 26](https://user-images.githubusercontent.com/219272/220967664-5d097380-1664-47fb-8937-5a28756ab704.png)

Reply:
updated and sort of works, but ends with  ERR_MISSING_SIG_ALG issue. This may be an authentik setup issue though